docs/getting-started/integrations/angular.md

```diff
@@ -690,6 +690,19 @@ export class AppComponent {
 
 For advanced usage see the {@link getting-started/setup/ui-language Setting the UI language} guide.
 
+<info-box warning>
+There is a known issue related to the localization in Angular 17. Read more in the [known issues section](#known-issues) below.
+</info-box>
+
+## Known issues
```

Comment:
I was thinking about adding a subsection, but in our docs, there is rarely a subsection for the known issues.

Reply:
And there is one known issue for now.

Reply:
I like the idea, and its present on some features docs so it's kind a template.